testregistry.textproto

```diff
@@ -759,6 +759,12 @@ test: {
   readme: ""
   exec: " "
 }
+test: {
+  id: "RT-3.33"
```

Comment:
Recommend PF-1.6. (if this test is to use policy-forwarding (pbr, filter based forwarding)

Reply:
Updated the same.